src/Nest/Aggregations/Metric/TopHits/TopHitsAggregation.cs

```diff
@@ -6,6 +6,7 @@ namespace Nest
 {
 	[JsonObject(MemberSerialization = MemberSerialization.OptIn)]
 	[ContractJsonConverter(typeof(AggregationJsonConverter<TopHitsAggregation>))]
+	[AggregateType(typeof(TopHitsAggregate))]
```

Comment:
I think we need to get rid of the json attributes now we have `AggregateType`. 

Maybe 


```
[AggregationJson(typeof(AggregationJson<TopHitsAggregation, TopHitsAggregate>))]
````